change/@fluentui-react-menu-bc30a5a7-6e96-41ab-a738-ea61bd34ef99.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "prerelease",
+  "packageName": "@fluentui/react-menu",
+  "email": "lingfan.gao@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
I am confused by the two change entries for a single package.

Reply:
yeah looks funny, gonna kill this one